packages/components/__tests__/script.tests.tsx

```diff
@@ -0,0 +1,75 @@
+import { act, create } from "react-test-renderer";
+import Script from "../script";
+
+describe("Script", () => {
+  test("should not render a script tag without code or src", () => {
+    let root;
+    act(() => {
+      root = create(<Script id="script-1">{"let index = 0;"}</Script>);
+    });
+
+    // This is `null`. This is here to make sure the virtual dom tree does not return anything.
+    expect(root.toJSON()).toMatchSnapshot();
```

Comment:
I would use an inline snapshot here. That way, we don't need to say "this is `null`", haha. :smile:

```suggestion
    // This is here to make sure the virtual dom tree does not return anything.
    expect(root.toJSON()).toMatchInlineSnapshot(`null`);
```

Reply:
Oh, such a good idea! 😄  Thank you!